babel.config.js

```diff
@@ -4,7 +4,7 @@ const packageJSON = require(path.join(__dirname, 'package.json'))
 
 // RedwoodJS targets Node.js 12.x because this is the default version
 // for Netlify's functions.
```

Comment:
Should update this comment as well. Perhaps link to docs where this can be found (both Vercel and Netlify).